frontend/dockerfile/dockerfile_lint_test.go

```diff
@@ -1010,15 +1010,8 @@ func checkProgressStream(t *testing.T, sb integration.Sandbox, lintTest *lintTes
 
 	f := getFrontend(t, sb)
 
-	attrs := lintTest.FrontendAttrs
-	if attrs == nil {
-		attrs = map[string]string{
-			"platform": "linux/amd64,linux/arm64",
```

Comment:
@daghack That was the original intention for this? I guess to check som warning deduplication. In this is needed then it should only be ignored on certain tests (although if it is needed then why isn't the same checked for unmarshal tests). Alternatively there should be some specific tests that specifically check for multi-platform cases.

Reply:
https://github.com/moby/buildkit/commit/88b2f70efa

It *seems* like it was just a way to keep an old default for the tests when a specific test didn't pass in specific frontend attributes. The old default came because I was emulating where I saw it elsewhere in tests, so there shouldn't be any problems with removing this.

Reply:
But do we have tests that cover deduplication on warnings. So that when user is building multiple platforms when they do not get the same warnings twice?